### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by hlaineka**
Navigation bar with dropdown menu on info.
Should be added as a component to App.tsx before routes. (how to hide on pages like register page?)
Tests run now with no complications, lint still needs some updating but there is a separete card about that

### CODE REVIEW

File: src/features/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </StyledNavDropdown>
+        <StyledNavLink to="/signout" className="navbar-link">
+          <div>Kirjaudu ulos</div>
+        </StyledNavLink>
+      </StyledNavBarItems>
+    </StyledNavBar>
+  );
+};
+
+export default NavigationBar;
```
[the remark this answers is not in the document]
Comment (rottabonus) [reply src/features/Navigation/Navigation.tsx:61]: Also, i dont think we need to have the _Styled_ prefix on these divs

File: src/custom.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
+	const value: any;
+	export = value;
+}
```
Comment (rottabonus) [inline src/custom.d.ts:4]: anys are scary :P

File: src/features/Navigation/Navigation.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (aeniinik) [reply src/features/Navigation/Navigation.tsx]: The navigation is now in smaller components.

**Review by rottabonus — APPROVED** [review]
Approved on the notion that Navigation.styles.ts will be removed and the styles inside will be sprinkled in corresponding component-files

**Pull request merged**